ansible/deploy_db.yml

```diff
@@ -36,7 +37,7 @@
     - hostTo: "{{ hot_standby_server }}"
     - state: 'present'
   tasks:
-    - import_tasks: roles/setup_auth_keys.yml
+    - include: roles/setup_auth_keys.yml
```

Comment:
`include` is deprecated, why didn't `import_tasks` work? If it needs to be dynamic then you should use `include_tasks`